components/job-orchestration/job_orchestration/scheduler/query/query_scheduler.py

```
@@ -890,7 +890,7 @@ async def handle_finished_search_job(
             new_job_status = QueryJobStatus.FAILED
             logger.error(
                 f"Search task job-{job_id}-task-{task_id} failed. "
-                f"Check {task_result.error_log_path} for details."
+                "Check container logs for details."
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)